src/index.js

```diff
@@ -1,53 +1,76 @@
-function createPropertyAssert(t, object, property) {
-  // assert(property in object);
+import * as t from 'babel-types';
+
+const ignoredNode = Symbol('ignoredNode');
+
+// returns t.memberExpression but marked to be ignored when visited
+function getIgnoredMemberExpression(object, property, computed) {
+  let node = t.memberExpression(object, property, computed);
+  node[ignoredNode] = true;
+  return node;
+}
+
+// require('../lib/safe_get.js')();
+function generateRequire() {
   return t.expressionStatement(
     t.callExpression(
-      t.identifier("assert"),
-      [t.binaryExpression(
-        "in",
-        property,
-        object
-      )]
+      t.callExpression(
+        t.identifier('require'),
+        [
+          t.stringLiteral('../lib/safe_get.js')
```

Comment:
I don't like '../lib' hardcoded part of the path. I'll think of what can we do about it.